ios/Podfile.lock

```diff
@@ -18,40 +20,52 @@ PODS:
     - GoogleToolboxForMac/Defines (= 2.1.1)
   - GoogleToolboxForMac/NSData+zlib (2.1.1):
     - GoogleToolboxForMac/Defines (= 2.1.1)
-  - Protobuf (3.3.0)
+  - nanopb (0.3.8):
+    - nanopb/decode (= 0.3.8)
+    - nanopb/encode (= 0.3.8)
+  - nanopb/decode (0.3.8)
+  - nanopb/encode (0.3.8)
+  - Protobuf (3.4.0)
```

Comment:
Curious if these upgrades (along with Firebase* libs upgrades) required by `RNDeviceInfo`?

Reply:
I don't actually know, @rasom might be able to answer that question